index.html

```diff
@@ -1,534 +1,1010 @@
+        </ul>
+      </nav>
 
-    <div class="header__title header__title-m">
-      <div class="header-container-m">
-        <h1>
-          Sebastian Gabriel
-          <!-- Green highlight effect -->
-          <span class="highlight">Nagy</span>
-        </h1>
-        <h4>Avocat definitiv Cluj-Napoca</h4>
-        <p>
-          <span class="header__desc">
-            Activitatea noastră este caracterizata de inovare și adaptabilitate la nevoile clientului, transparență,
-            atenție la detalii, experiență și expertiza de înaltă calitate și nu în ultimul rand, disponibilitate în
-            relația cu clienții.
-          </span>
-        </p>
+      <div class="header__title">
+        <div class="header-container">
+          <h1>
+            Sebastian Gabriel
+            <!-- Green highlight effect -->
+            <span class="highlight">Nagy</span>
+          </h1>
+          <h4>Avocat definitiv Cluj-Napoca</h4>
+          <p>
+            <span class="header__desc">
+              Activitatea noastră este caracterizata de inovare și
+              adaptabilitate la nevoile clientului, transparență, atenție la
+              detalii, experiență și expertiza de înaltă calitate și nu în
+              ultimul rand, disponibilitate în relația cu clienții.
+            </span>
+          </p>
+        </div>
         <!-- <button class="btn--text btn--scroll-to">Mai mult &DownArrow;</button> -->
-        <a class="btn--show-modal" href="#contact">Contactează</a>
+        <button class="btn--show-modal btn--text">Contactează</button>
 
+        <!--TODO: smaller image, rounded borders -->
+        <img
+          src="img/hero.jpg"
+          class="header__img"
+          alt="Avocat Sebastian Nagy imagine profil | Avocat definitiv Cluj-Napoca"
+        />
       </div>
-      <img src="img/hero.jpg" class="header__img"
-        alt="Avocat Sebastian Nagy imagine profil | Avocat definitiv Cluj-Napoca" />
-    </div>
-  </header>
-
-  <section class="section" id="despre">
-    <div class="section__title">
-      <h2 class="section__description">Despre</h2>
-      <h3 class="section__header">
-        Abordarea noastră în domeniul dreptului
-      </h3>
-    </div>
 
-    <div class="operations">
-      <div class="operations__tab-container">
-        <button class="btn operations__tab operations__tab--1 operations__tab--active" data-tab="1">
-          Determinare
-        </button>
-        <button class="btn operations__tab operations__tab--2" data-tab="2">
-          Abilitate
-        </button>
-        <button class="btn operations__tab operations__tab--3" data-tab="3">
-          Pasiune
-        </button>
-      </div>
-      <div class="operations__content operations__content--1 operations__content--active">
-        <div class="operations__icon operations__icon--1">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-upload"></use>
-          </svg>
+      <div class="header__title header__title-m">
```

Comment:
Do you close the `</header>` element anywhere? It seems like you removed some closing tags and this might introduce bugs, please double-check.